compiler/rustc_const_eval/src/const_eval/machine.rs

```diff
@@ -34,21 +34,7 @@ impl<'mir, 'tcx> InterpCx<'mir, 'tcx, CompileTimeInterpreter<'mir, 'tcx>> {
         // All `#[rustc_do_not_const_check]` functions should be hooked here.
         let def_id = instance.def_id();
 
-        if Some(def_id) == self.tcx.lang_items().const_eval_select() {
```

Comment:
why was this removed?

Reply:
Only compile-time `const_eval_select` is defined in core::intrinsics. If you call it is in runtime context, rustc_mir_transform will replace it with the call to the runtime argument.

Reply:
🤦 ok I completely blanked here, thanks!